qt/widgets/spectrumviewer/src/SpectrumView.cpp

```diff
@@ -248,6 +262,15 @@ void SpectrumView::respondToTabCloseReqest(int tab) {
   }
 }
 
+void SpectrumView::selectData(int spectrumNumber, double dataVal) const {
```

Comment:
I cannot recall, is this needed for something? 

Also, that `const` qualifier is a bit misleading... this method does modify internal state via `setHGraph`, etc. It only gets away with it because `m_spectrumDisplay` is a list of pointers... which don't themselves get modified, but what they point to does. 

Technically `const`ness has not been violated, but conceptually the state of what the class instance represents has changed. In fact, the express purpose of this method is to change its state.